stories/form.stories.tsx

```diff
@@ -4,16 +4,21 @@ import { action } from '@storybook/addon-actions';
 
 import Form from '../src/components/Form';
 
-const multiLineFieldset =  (
+const multiLineFieldset = (id: number) => (
   <fieldset className="well">
     <legend>Sample fieldset</legend>
-    <input placeholder="A sample field"></input>
-    <input placeholder="Another sample field"></input>
+    <label htmlFor={`input-${id}`}>Sample 1</label>
```

Comment:
This was tedious but just to not have duplicate ID issues in the browser for the Storybook demo.